packages/ui/cypress/support/constants.ts

```diff
@@ -20,6 +20,9 @@ export const constants = {
     events: {
         selectWord: 'Test_SelectWord',
     },
+    strings: {
+        customCallbackName: 'Enter name for callback',
```

Comment:
Feels like this should be something like "New Callback" or "New Mock Callback"

Reply:
I'm not sure what you mean. The key is for whatever option allows giving custom callback names, and the value has to match the actual option name for the tests to select it.

If you wanted a different option name I think that's defined in the action creator modal.
I think those proposed names are misleading. It's not creating a new callback, those can only defined in code. You're creating a new action and giving a temporary name until it's defined. Also don't want to confuse people with the word mock. Mock results is already poor generic name so we want to only use those there.